src/main/java/org/broadinstitute/consent/http/models/Study.java

```diff
@@ -15,9 +15,9 @@ public class Study {
   private Boolean publicVisibility;
   private String piName;
   private List<String> dataTypes;
-  private Set<Integer> datasetIds;
+  private Set<Integer> datasetIds = new HashSet<>();
   private Set<Dataset> datasets;
-  private Set<StudyProperty> properties;
+  private Set<StudyProperty> properties = new HashSet<>();
```

Comment:
I think this is a good opportunity to clean this up a bit more. I'd also make these fields final (so a caller can't overwrite them with a `null` value) and initialize `datasets` here and make it `final` too.